Refactor some workflow editor plumbing to VueJS

jmchilton opened the pull request:
- Move center panel construction into VueJS object.
- Construct ``WorkflowView`` object in VueJS instead of in mako.
- Migrate mako CSS to VueJS - scoping some of it (canvas stuff can't be yet).

jmchilton committed:
Refactor editor canvas to Vue.

jmchilton committed:
Refactor workflow editor CSS to Vue (scoping what works).

jmchilton committed:
Refactor WorkflowView creation out of mako.

jmchilton committed:
prettier new workflow editor Vue code.

guerler commented on client/galaxy/scripts/components/Workflow/Editor/WorkflowEditor.vue:
```diff
@@ -0,0 +1,149 @@
+                        ></canvas>
+                        <div
+                            id="overview-viewport"
+                            style="position: absolute; width: 0px; height: 0px; border: solid blue 1px; z-index: 10;"
```
Can we move these styles to the scoped styles section.

jmchilton replied:
I've refactored all embedded CSS out of this file.

jmchilton committed:
Rework MarkdownEditor component usage.

- More robust CSS sizing (get rid of awful height: 95% hack).
- Rework JS so component injection of markdown editor can be used.
- Replace jQuery hide/show interactions for buttons, etc.. with v-show.

jmchilton committed:
Refactor all embedded styles out of WorkflowEditor.

jmchilton committed:
Migrate wf editor run jQuery handling to Vue.

jmchilton committed:
Migrate wf editor save jQuery handling to Vue.

jmchilton committed:
Migrate wf editor reportHelp jQuery handling to Vue.

jmchilton committed:
Refactor wf editor setting menu construct into Vue...

... sortof anyway.
